### PR TITLE
Use rimraf package instead of rm -rf commands

### PR DESCRIPTION
**Pull request by pvanaarde**
Update the build-css and clean commands to be cross-platform capable rather than UNIX specific. Changed to use "rimraf" rather than "rm -rf".

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "react-redux": "^5.0.3",
     "redux": "^3.6.0",
     "redux-observable": "^0.13.0",
+    "rimraf": "^2.6.1",
```
Comment (danmarshall) [inline package.json]: can you move this to the devDependencies section below?

**Comment by GeekTrainer** [comment]
Bumping @pvanaarde WRT the update requested by @danmarshall.

Although I am curious the impact of leaving it with `rm -rf`. I personally run everything via a Git shell, so the command executes just fine. I have this thing about trying to avoid additional dependencies. :-)

**Comment by billba** [comment]
@GeekTrainer what is a "Git shell"?

**Comment by GeekTrainer** [comment]
@billba A PowerShell plugin provided with the GitHub client on Windows.

**Pull request merged**